plugins/package-curation-providers/file/src/main/kotlin/FilePackageCurationProvider.kt

```diff
@@ -103,7 +103,7 @@ class FilePackageCurationProvider(
                     "The path '$it' does not exist."
                 }
 
-                if (it.isDirectory) FileFormat.findFilesWithKnownExtensions(it) else listOf(it)
+                if (it.isDirectory) FileFormat.findFilesWithKnownExtensions(it).sorted() else listOf(it)
```

Comment:
We had this topic of ordering vs. added complexity already before, but I do not recall the details.

My preference still is to not assume any ordering but instead require each curation to be correct on its own.
Because for correct curations, the order does not matter. Anyhow, it'd be interested to hear further opinions on this. Also how much of an edge case this is. IMO it'd would be that problematic to not use namespace curations 
if the curation does not apply to all packages in the namespace.

Reply:
> I'm happy to drop the last commit.

Done.